### ISSUE TITLE
Variant data under Configurable product is missing Image Url

### ISSUE THREAD

**Issue by nrkapoor** [magento/catalog-storefront#52]
Variant data under the Configurable product is missing Image Url.

In some situations, the Variant data is missing altogether - https://jira.corp.magento.com/browse/SEARCH-134 from the configurable product, however, this is a separate issue.

In this case, even if Variant data is available, we do not have the image URL of the variant data.

This is needed if we want the ability to show the variant image if the query term matches the variant of the configurable product, e.g. searching for "pants women red" on Luma should show us women red pants.

Notes
* After #19 is complete, need to make sure feed contains product image URLs
